### PR TITLE
[W-14887715] Adding links to documentation site

### PR DESCRIPTION
**Pull request by nledesma**

### CODE REVIEW

**Review by balbifm — COMMENTED** [review]
Two things:
* I would like a link to the examples as well
* Is there a way to not depend on the **_1.0_** in the url path for the links? @glenn-rodgers-sf

File: readme-template.md [hunk 1]
```diff
@@ -1,19 +1,32 @@
 # "{{ project-name }}" Policy
 
-## Development lifecycle
+This policy was created with the Policy Development Kit. For more information about the PDK, visit the [Documentation site](https://docs.mulesoft.com/pdk/1.0/policies-pdk-overview).
```
Comment (glenn-rodgers-sf) [inline readme-template.md:3]:
```suggestion
This policy was created with the Flex Gateway Policy Development Kit (PDK). To find the complete PDK documentation, see [PDK Overview](https://docs.mulesoft.com/pdk/1.0/policies-pdk-overview) on the Mulesoft documentation site.
```

File: readme-template.md [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
+
+
+## Make command reference
 This project has a Makefile that includes different goals that assist the developer during the policy development lifecycle.
 
+*For more information about the Makefile, visit the [Makefile](https://docs.mulesoft.com/pdk/1.0/policies-pdk-create-project#makefile) docs section.*
```
Comment (glenn-rodgers-sf) [inline readme-template.md:9]:
```suggestion
*For more information about the Makefile, see [Makefile](https://docs.mulesoft.com/pdk/1.0/policies-pdk-create-project#makefile).*
```

File: readme-template.md [hunk 3]
```diff
@@ -1,19 +1,32 @@
...
+
 ### Setup
 The `make setup` goal installs the Policy Development Kit internal dependencies for the rest of the Makefile goals.
 Since these dependencies are provided by the Anypoint Platform, it requires the user to be authenticated with a set of valid Anypoint credentials.
 
+*For more information about `make setup`, visit the [Setup the PDK Build environment](https://docs.mulesoft.com/pdk/1.0/policies-pdk-create-project#setup-the-pdk-build-environment) docs section.*
```
Comment (glenn-rodgers-sf) [inline readme-template.md:15]:
```suggestion
*For more information about `make setup`, see [Setup the PDK Build environment](https://docs.mulesoft.com/pdk/1.0/policies-pdk-create-project#setup-the-pdk-build-environment).*
```

File: readme-template.md [hunk 4]
```diff
@@ -1,19 +1,32 @@
...
+
 ### Build asset files
 The `make build-asset-files` goal generates all the policy asset files required to build, execute, and publish the policy. This command also updates the `config.rs` source code file with the latest configurations defined in the policy definition.
 
+*For more information about creating a policy definition, visit the [Defining a Policy Schema Definition](https://docs.mulesoft.com/pdk/1.0/policies-pdk-create-schema-definition) docs section.*
```
Comment (glenn-rodgers-sf) [inline readme-template.md:20]:
```suggestion
*For more information about creating a policy definition, see [Defining a Policy Schema Definition](https://docs.mulesoft.com/pdk/1.0/policies-pdk-create-schema-definition).*
```

File: readme-template.md [hunk 5]
```diff
@@ -1,19 +1,32 @@
...
+
+*For more information about `make build-asset-files`, visit the [Compiling Custom Policies](https://docs.mulesoft.com/pdk/1.0/policies-pdk-compile-policies) docs section.*
```
Comment (glenn-rodgers-sf) [inline readme-template.md:22]:
```suggestion
*For more information about `make build-asset-files`, see [Compiling Custom Policies](https://docs.mulesoft.com/pdk/1.0/policies-pdk-compile-policies).*
```

File: readme-template.md [hunk 6]
```diff
@@ -1,19 +1,32 @@
...
+
 ### Build
 The `make build` goal compiles the WebAssembly binary of the policy.
 Since the source code must be in sync with the policy definition configurations, this goal runs the `build-asset-files` before compiling.
 
+*For more information about `make build`, visit the [Compiling Custom Policies](https://docs.mulesoft.com/pdk/1.0/policies-pdk-compile-policies) docs section.*
```
Comment (glenn-rodgers-sf) [inline readme-template.md:28]:
```suggestion
*For more information about `make build`, see [Compiling Custom Policies](https://docs.mulesoft.com/pdk/1.0/policies-pdk-compile-policies).*
```

File: readme-template.md [hunk 7]
```diff
@@ -28,11 +41,15 @@ Otherwise, to complete the registration we recommend using the Anypoint Platform
 
 The `playground/config` directory can also contain other resource definitions, such as accessory services used by the policy (Eg. a remote authentication service).
 
+*For more information about `make run`, visit the [Debugging Custom Policies Locally with PDK](https://docs.mulesoft.com/pdk/1.0/policies-pdk-debug-local) docs section.*
```
Comment (glenn-rodgers-sf) [inline readme-template.md:44]:
```suggestion
*For more information about `make run`, see [Debugging Custom Policies Locally with PDK](https://docs.mulesoft.com/pdk/1.0/policies-pdk-debug-local).*
```

File: readme-template.md [hunk 8]
```diff
@@ -28,11 +41,15 @@ Otherwise, to complete the registration we recommend using the Anypoint Platform
...
+
 ### Test
-The `make test` goal runs unit tests and integration tests. Integration tests are 
+The `make test` goal runs unit tests and integration tests. Integration tests are
 placed in the `tests` directory and are configured with the files placed at the
 `tests/<module-name>/<test-name>` directory.
```
Comment (glenn-rodgers-sf) [inline readme-template.md:49]: These should all be on the same line right?

File: readme-template.md [hunk 9]
```diff
@@ -28,11 +41,15 @@ Otherwise, to complete the registration we recommend using the Anypoint Platform
...
 
+*For more information about writing integration tests, visit the [Writing Integration Tests](https://docs.mulesoft.com/pdk/1.0/policies-pdk-integration-tests) docs section.*
```
Comment (glenn-rodgers-sf) [inline readme-template.md:51]:
```suggestion
*For more information about writing integration tests, see [Writing Integration Tests](https://docs.mulesoft.com/pdk/1.0/policies-pdk-integration-tests).*
```

File: readme-template.md [hunk 10]
```diff
@@ -42,5 +59,9 @@ Since the publish goal is intended to publish a policy asset in development, the
 - assetId: _{your-policy-asset-id}-dev_
 - version: _{your-policy-version}-20230618115723_
 
+*For more information about publishing policies, visit the [Uploading Custom Policies to Exchange](https://docs.mulesoft.com/pdk/1.0/policies-pdk-publish-policies) docs section.*
```
Comment (glenn-rodgers-sf) [inline readme-template.md:62]:
```suggestion
*For more information about publishing policies, visit the [Uploading Custom Policies to Exchange](https://docs.mulesoft.com/pdk/1.0/policies-pdk-publish-policies) docs section.*
```
```suggestion
*For more information about publishing policies, see [Uploading Custom Policies to Exchange](https://docs.mulesoft.com/pdk/1.0/policies-pdk-publish-policies).*
```

File: readme-template.md [hunk 11]
```diff
@@ -42,5 +59,9 @@ Since the publish goal is intended to publish a policy asset in development, the
...
+
 ### Release
 The `make release` goal also publishes the policy to Anypoint Exchange, but as a ready for production asset. In this case, the groupId, visible name, assetId and version will be the ones defined in the project.
+
+*For more information about releasing policies, visit the [Uploading Custom Policies to Exchange](https://docs.mulesoft.com/pdk/1.0/policies-pdk-publish-policies) docs section.*
```
Comment (glenn-rodgers-sf) [inline readme-template.md:67]:
```suggestion
*For more information about releasing policies, see [Uploading Custom Policies to Exchange](https://docs.mulesoft.com/pdk/1.0/policies-pdk-publish-policies).*
```

File: readme-template.md (hunk 8 above)
Comment (nledesma) [reply readme-template.md:48]: yes! fixed!

**Review by JaatadiaMulesoft — APPROVED** [review]

**Pull request merged**